tests/suites/test_suite_platform.function

```diff
@@ -76,6 +74,13 @@ void time_delay_milliseconds(int delay_ms)
 /* END_CASE */
 
 /* BEGIN_CASE depends_on:MBEDTLS_HAVE_TIME */
+
+/*
+ * WARNING: DONOT ENABLE THIS TEST. RESERVE IT HERE TO KEEP THE REASON.
```

Comment:
Minor: English:
```suggestion
 * WARNING: DO NOT ENABLE THIS TEST. We keep the code here to document the reason.
```

Reply:
Yes, could this change be made, then I will approve